### PR TITLE
validate.lic: Add cambrinth validator for crafting, fixup training_sp…

### PR DESCRIPTION
**Pull request by Sarvatt**
…ells check.

data['name'] isn't populated but data['abbrev'] works

### CODE REVIEW

**Comment by Sarvatt** [comment]
would data['name'] be populated after the dependency update adding it to transform_settings?

**Comment by Sarvatt** [comment]
nope it isn't, [validate: WARNING:<  in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.  >], abbrev works.

File: validate.lic [hunk 1]
```diff
@@ -109,12 +109,19 @@ class DRYamlValidator
...
             .reject { |skill_name| valid_crafting_training_spells_skills.include?(skill_name) }
-            .each { |skill_name| error("Invalid crafting_training_spells: skill name '#{skill_name}'. Valid skill names are '#{valid_crafting_training_spell_skills}'") }
+            .each { |skill_name| error("Invalid crafting_training_spells: skill name '#{skill_name}'. Valid skill names are '#{valid_crafting_training_spells_skills}'") }
+  end
+
+  def assert_that_crafting_training_spells_have_no_cambrinth(settings)
+    return unless settings.crafting_training_spells
+
+    settings.crafting_training_spells.select { |_spell, data| data['cambrinth'] }
+            .each { |_spell, data| warn("#{data['abbrev']} in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.") }
```
Comment (rpherbig) [inline validate.lic]: ❓ I think `spell` (the unused first arg) might be the full spell name, which might be better to print than the abbrev. What do you think?

Comment (Sarvatt) [reply validate.lic]: That looks better

[validate: WARNING:< Augmentation in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.  >]

**Comment by Sarvatt** [comment]
```
crafting_training_spells:
  Augmentation:
    abbrev: WOTP
    symbiosis: true
    mana: 15
    cambrinth:
      - 10
  Warding:
    abbrev: ey
    symbiosis: true
    mana: 15
  Utility:
    abbrev: SKS
    symbiosis: true
    mana: 15

**Pull request merged**